plugins/inventory/hrobot.py

```diff
@@ -0,0 +1,180 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Ansible, but is an independent component.
+# This particular file snippet, and this file snippet only, is BSD licensed.
+# Modules you write using this snippet, which is embedded dynamically by Ansible
+# still belong to the author of the module, and may assign their own license
+# to the complete work.
+#
+# Copyright (c), Felix Fontein <felix@fontein.de>, 2019
+#
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = r"""
+    name: hrobot
+    plugin_type: inventory
+    author:
+        - Oleksandr Stepanov
+    short_description: Hetzner Robot inventory source
+    requirements:
+        - python >= 3.9
```

Comment:
Please remove this. There is no reason to restrict the Python version to 3.9+. Everything else in the collection suports 2.7+, so this code should too.